### DISCUSSION TITLE
iOS app doesn't connect to local server

### DISCUSSION THREAD

**Discussion by alexandru-andronache** [TomBursch/kitchenowl#249] [Q&A]
Hi,

I've install the app on my local synology machine using docker and it's working great locally but when i try to use the iOS app it doesn't connect. I've tried both quick connect and just use the local IP address but it doesn't work. With quick connect it doesn't find the server and if i try with the local IP address the app enters an infinite loop. Did i do something wrong?

Alex.

**Comment by TomBursch** [comment]
Hi, thanks for reaching out.
I'm not sure what quick connect is, but can you connect to the web app using safari? Do you run a reverse proxy in front of KitchenOwl? 

Thanks, Tom
